utils/led.c

```diff
@@ -6,8 +6,8 @@
  */
 
 #include <stdint.h>
-#include <stdio.h>
-#include <stdlib.h>
+//#include <stdio.h>
```

Comment:
Please remove dead code, Git should have us covered.

Reply:
Removed.